wasm/js-canvas.go

```diff
@@ -25,24 +25,27 @@ func GetCanvas(domElementId string) Canvas {
 // Notes:
 // Consider drawImage, if we can generate the correct format to output in 1 call.
 // https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/drawImage
-// Also find out if we can do double buffering with canvas
+// OR
+// access the canvas pixel data directly might be faster - at least for unscaled output
+//  var id = ctx.getImageData(0, 0, canvasWidth, canvasHeight)
+//  var pixels = id.data
+//  pixels[0] = 0
+//  pixels[1] = 255
+//  pixels[2] = 255
+//  pixels[3] = 255
 
 func (c Canvas) SetColor(color anotherworld.Color) {
-	c.fillStyle(getHexColorFrom(color))
+	c.context2d.Set("fillStyle", fmt.Sprintf("rgb(%d, %d, %d)", color.R, color.G, color.B))
```

Comment:
do you think the sprintf here needs to be optimized? Not sure what it does under the hood.

Reply:
looking at the performance its more about the fill Style call. I have an idea how to optimize in a later pr

Reply:
Sounds good, I also have some links in https://github.com/neophob/ganother-world/issues/30 I didn't get around to test